client/a8c-for-agencies/sections/partner-directory/lib/agency-directory-application-form.tsx

```diff
@@ -0,0 +1,26 @@
+import { AgencyDirectoryApplication } from '../types';
+
+/*
+ * Check if the form has services selected
+ */
+export function hasServices( formData: AgencyDirectoryApplication ): boolean {
+	return formData.services.length > 0;
+}
+
+/*
+ * Check if the form has products selected
+ */
+export function hasProducts( formData: AgencyDirectoryApplication ): boolean {
+	return formData.products.length > 0;
+}
+
+/*
+ * Check if the form has directories selected
+ */
+export function hasDirectory( formData: AgencyDirectoryApplication ): boolean {
+	return formData.directories.length > 0;
+}
+
+export function isValidApplicationForm( formData: AgencyDirectoryApplication ): boolean {
+	return hasServices( formData ) && hasProducts( formData ) && hasDirectory( formData );
+}
```

Comment:
Really do we need a library for this?

Reply:
I'm going to take over this PR and add a commit

Reply:
@cleacos, good point. I wasn't sure really where to put this. But yeah `agency-expertise/index.tsx` looks good to me 👍